fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml

```diff
@@ -61,19 +63,32 @@ This job performs the Sanity Check on the region given as parameter:&#xd;
   <customWorkspace>$FIHEALTH_WORKSPACE</customWorkspace>
   <builders>
     <hudson.tasks.Shell>
-      <command># Execute &apos;exec&apos; script
-./$FIHEALTH_SANITY_PROJECT/resources/scripts/jenkins.sh exec
+      <command># Execute &amp;apos;exec&amp;apos; script
+./$FIHEALTH_SANITY_PROJECT/resources/scripts/jenkins.sh exec</command>
+    </hudson.tasks.Shell>
+    <hudson.tasks.Shell>
+      <command># WORKAROUND - Publish data into Master CB.
+
+MASTER_ADAPTER_URL=http://10.0.64.4:1337
```

Comment:
I suggest to leave it as it due to this is the script (workaround) to execute. It is just the definition of the variable to parametrize the curl operation.